cpp/log/cert_checker_test.cc

```diff
@@ -273,7 +273,8 @@ TEST_F(CertCheckerTest, PreCert) {
   checker_.LoadTrustedCertificates(cert_dir_ + "/" + kCaCert);
   EXPECT_OK(checker_.CheckPreCertChain(&chain, &issuer_key_hash, &tbs));
   string expected_key_hash;
-  ASSERT_EQ(Cert::TRUE, chain.CertAt(1)->SPKISha256Digest(&expected_key_hash));
+  ASSERT_EQ(util::Status::OK,
```

Comment:
There's `ASSERT_OK` and `EXPECT_OK` in `util/status_test_util.h`, could you use those instead?